internal/deploy/kustomize/daily/examples/watch-another-namespace/wlo-ns2/kustomization.yaml

```diff
@@ -7,8 +7,8 @@ resources:
 - ../../../overlays/watch-another-namespace/wlo-ns
 patches:
 - path: wlo-deployment.yaml
-- path: role-patch1.yaml
-- path: role-patch2.yaml
-- path: role-patch3.yaml
-- path: role-patch4.yaml
+- path: websphere-liberty-leader-election-role.yaml
```

Comment:
@kabicin I believe the naming of this is what you brought up in the scrum. Although I agree that it would be ideal to align the naming with OLO/RCO, I think we can leave the WLO names as is, primarily to avoid affecting users upgrading from a previous release (so that resources with old names are not discarded)